src/dao/pages/DAOProposals/CreateDAOProposal.tsx

```diff
@@ -61,12 +67,11 @@ export function CreateDAOProposal() {
   const currentDaoType = location.pathname.split("/").at(1) ?? "";
   const currentWizardStep = getLastPathInRoute(location.pathname);
   const backTo = `/${currentDaoType}/${daoAccountId}/${Routes.Overview}`;
-  const daosQueryResults = useDAOs(daoAccountId);
   const handleTransactionSuccess = useHandleTransactionSuccess();
-  const { data: daos } = daosQueryResults;
-  const dao = daos?.find((dao) => dao.accountId === daoAccountId);
   const isNotFound = daosQueryResults.isSuccess && isNil(dao);
-  const { ownerIds, safeId: safeAccountId = "", threshold, safeEVMAddress } = (dao as MultiSigDAODetails) ?? {};
+  const { ownerIds, threshold, safeEVMAddress } = (dao as MultiSigDAODetails) ?? {};
+  const daoSafeIdQueryResults = useContract(safeEVMAddress ?? "");
```

Comment:
What happens if `useContract` is called with an empty string `""` ? Will an error be thrown, will the app fail, etc.?

Reply:
No. as in the enable condidtion of hook I have added the check for empty string so no error will be thrown.